api/src/main/java/uk/gov/dhsc/htbhf/claimant/entitlement/EntitlementCalculator.java

```diff
@@ -41,41 +43,52 @@ public EntitlementCalculator(
         this.voucherValueInPence = voucherValueInPence;
     }
 
-    public VoucherEntitlement calculateVoucherEntitlement(Claimant claimant, List<LocalDate> childrenDatesOfBirth) {
-        int numberOfChildrenUnderFour = getNumberOfChildrenUnderFour(childrenDatesOfBirth);
-        int numberOfChildrenUnderOne = getNumberOfChildrenUnderOne(childrenDatesOfBirth);
+    /**
+     * Calculates the {@link VoucherEntitlement} a claimant is entitled to for a given date. The number of children under one and
```

Comment:
This JavaDoc isn't completely accurate anymore as its given the DoB of all children.

Reply:
It is, but the comment is still correct. We are calculating the number of children under one and four.